packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -320,4 +322,11 @@ Will double processing times. Use only for debugging purposes.",
     default: defaultOptions.chain.pruneHistory,
     group: "chain",
   },
+
+  "chain.noValidatorCustody": {
+    type: "boolean",
+    description: "Disable dynamic updates to validator custody requirements",
+    defaultDescription: "false",
```

Comment:
Is `defaultDescription` needed? Should we default it to `true`?

Reply:
not needed but fine to leave that there